inspirehep/modules/orcid/schema.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+import dojson
+
+from dojson import utils
+
+from inspirehep.utils.record import get_abstract, get_subtitle, get_title
+
+orcid_overdo = dojson.Overdo()
+
+
+@orcid_overdo.over('title', 'titles')
+def title_rule(self, key, value):
+    title = get_title({"titles": value})
+    if title == '':
+        raise KeyError
+    subtitle = ""
+    try:
+        subtitle = get_subtitle({"titles": value})
+    except KeyError:
+        pass
+    return {"title": title,
+            "subtitle": subtitle}
+
+
+@orcid_overdo.over('journal_title', 'publication_info')
+def publication_rule(self, key, value):
+    try:
+        return value[0].get('journal_title')
+    except (TypeError, KeyError):
+        pass
+
+
+@orcid_overdo.over('short_description', 'abstracts')
+def abstract_rule(self, key, value):
+    try:
+        return get_abstract({"abstracts": value})
+    except (TypeError, KeyError):
+        pass
+
+
+@orcid_overdo.over('publication_date', 'imprints')
```

Comment:
Yeah that was my exact reaction :D It throws an `error: bad character in group name 'Ijournal-title'`